docs/_guide/03-template-reference.md

```diff
@@ -309,9 +309,14 @@ const myTemplate = () => html`
 
 ### guard
 
-`guard(expression, valueFn)`
+`guard(expressions, valueFn)`
```

Comment:
I think `values` makes more sense as the first argument, as the function doesn't actually receive expressions, but the results of those expressions as values. I just called the parameters `value` ('cause it can be just one object) and `f` in the code.

Reply:
OK. Would it be correct to call the function a template function? Maybe just call it templateFunction here?

Reply:
The second argument (`valueFn`) is a function that returns any arbitrary value that lit-html can render, which includes templates, but also strings, arrays, etc. I think we generally use `value` to describe this type of object throughout lit-html.

Reply:
Thanks @ruphin. Yep, I was just checking on this.

`guard(value, valueFn)` is kind of confusing, but if we try to make it less vague, we end up with `guard(valueOrValuesToMonitor, functionThatReturnsSomethingToRender)` which is just ridiculous ...

Reply:
I like `dependencies` as the first argument, fwiw.  They are what the `valueFn` depends on.